src/eureka/S3_data_reduction/optspex.py

```diff
@@ -5,6 +5,38 @@
 from . import plots_s3
 
 
+def standard_spectrum(data, apdata, aperr):
+    """Compute the standard box spectrum.
+
+    Parameters
+    ----------
+    data : Xarray Dataset
+        The Dataset object.
+    apdata : ndarray
+        The pixel values in the aperture region.
+    aperr : ndarray
+        The noise values in the aperture region.
+
+    Returns
+    -------
+    data : Xarray Dataset
+        The updated Dataset object in which the spectrum data will stored.
+    """
+    data['stdspec'] = (['time', 'x'], np.sum(apdata, axis=1))
+    data['stdvar'] = (['time', 'x'], np.sum(aperr ** 2, axis=1))
+    data['stdspec'].attrs['flux_units'] = \
+        data.flux.attrs['flux_units']
+    data['stdspec'].attrs['time_units'] = \
+        data.flux.attrs['time_units']
+    data['stdvar'].attrs['flux_units'] = \
+        data.flux.attrs['flux_units']
+    data['stdvar'].attrs['time_units'] = \
+        data.flux.attrs['time_units']
+    # FINDME: stdvar >> stdspec, which is a problem
```

Comment:
This issue has been resolved now?

Reply:
Not sure, I still see: `# FINDME: stdvar >> stdspec, which is a problem`

Reply:
Sorry, I meant is that comment no longer valid? I previously suggested we remove that comment, and you said that was still happening so it should be left

Reply:
Glad I can be contradictory in my statements, depending on my mood. Might as well flip a coin.

Reply:
Alright, well the calculation that would be messing up stdspec is likely the DN->e- conversion, so I'll remove this comment from here since it isn't in a relevant spot anyway, and if we do find that this is an issue we can make a GitHub Issue about it